packages/kbn-apm-config-loader/src/config.ts

```diff
@@ -267,7 +289,7 @@ export class ApmConfiguration {
    * Reads APM configuration from different sources and merges them together.
    */
   private getConfigFromAllSources(): AgentConfigOptions {
-    const configFromKibanaConfig = this.getConfigFromKibanaConfig();
+    const { servicesOverrides, ...configFromKibanaConfig } = this.getConfigFromKibanaConfig();
```

Comment:
Do we need to set a flag in Serverless base config to easily switch between RUM enabled that defaults to 'false'? when we add per-project overrides, we could move to an override to change whatever default there is for Kibana Serverless.

Reply:
Happy to add any "default" settings to our `config/serverless.yml` if you think it's worth it.

I didn't add it because, IMO, it's an environment-specific setup. IMO, the air-gapped situation is the orchestration to deal with. If the controllers solve this issue, we shouldn't need to ship a new version of Kibana to re-enable the RUM agent.

But again: no strong opinion here.